lib/srv/discovery/fetchers/db/aws_elasticache.go

```diff
@@ -252,3 +252,16 @@ func getElastiCacheResourceTags(ctx context.Context, client elasticacheiface.Ela
 
 	return output.TagList, nil
 }
+
+// MatchingLabels returns the labels that the fetcher is matching.
+func (f *elastiCacheFetcher) MatchingLabels() types.Labels {
+	return f.cfg.Labels
+}
+
+func (f *elastiCacheFetcher) MatchesResource(r types.ResourceWithLabels) bool {
+	db, ok := r.(types.Database)
+	if !ok {
+		return false
+	}
+	return db.IsElastiCache() && !db.GetAWS().IsEmpty()
```

Comment:
nit: I don't think`!db.GetAWS().IsEmpty()` is needed. same for other database types